src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -113,6 +120,51 @@ export default {
 		menuOpen: {
 			type: Boolean,
 			default: false
+		},
+		/**
+		 * Refers to the icon on the left, this prop accepts a class
+		 * like 'icon-category-enabled' or an absolute URL. If you
+		 * pass in an URL the component will display the linked image;
+		 */
+		icon: {
+			type: String,
+			required: false
+		},
+		/**
+       	* Allows you to choose wether you want a parent element to always
+	   	* display it's children (true) or give the possibility to
+		* collapse them into one element (false).
+       	*/
+		expanded: {
+			type: Boolean,
+			default: true
+		},
+		/**
+		* Passing in a HEX color string in this prop will hide any
+		* icon or image and display a bullet with the selected
+		* colors instead.
+       	*/
+		bullet: {
+			type: String,
+			default: ''
```

Comment:
This could use a validator function :)
```suggestion
			default: '',
			validator: value => {
				return /^([a-f0-9]{3}){1,2}$/i.test(value)
			}
```